pkg/schedule/schedulers/grant_leader.go

```diff
@@ -178,6 +178,24 @@ func (s *grantLeaderScheduler) EncodeConfig() ([]byte, error) {
 	return EncodeConfig(s.conf)
 }
 
+func (s *grantLeaderScheduler) ReloadConfig() error {
```

Comment:
Can we reduce the duplication in ReloadConfig?

Reply:
Different config structures need to have different implementation, I think it's OK to have some duplication here.